doc/whats_new/v0.20.rst

```diff
@@ -21,6 +21,10 @@ occurs due to changes in the modelling logic (bug fixes or enhancements), or in
 random sampling procedures.
 
 - :mod:`sklearn.neighbors` when ``metric=='jaccard'`` (bug fix)
+- :class:`metrics.pairwise_distances` when ``n_jobs > 1`` and
```

Comment:
I hadn't expected this summary to include this much detail. Just say "use of 'seuclidean' or 'mahalanobis' metrics in some cases"..?

Reply:
Don't need to pander to me, but there's a proper list of changes below. This is only meant to be a summary, and really is not that important for regressions in minor versions.

Reply:
The point is also that it applies beyond just users of metrics.